.github/actions/create-venv-for-tests/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
         python -m pip uninstall jedi --yes
         python -m pip install jedi==0.17.2
         python -m pip install pandas
-        python -m pip install ipywidgets
+        python -m pip install ipywidgets==7.7.2
```

Comment:
Was causing widget tests to fail, as IPyWidget 8.0 was getting installed (which isn't supported, not yet)